flashinfer/gemm/kernels/grouped_gemm_masked_blackwell.py

```
@@ -1746,41 +1980,82 @@ def kernel(
-                                # The original c_pipeline.producer_acquire()
-                                #   := PipelineTmaStore.producer_acquire()
-                                #   := TmaStoreFence.wait()
-                                #   := cute.arch.cp_async_bulk_wait_group(self.num_stages - 1, read=True)
-                                cute.arch.cp_async_bulk_wait_group(
-                                    self.num_c_stage - 1,
-                                    # Change `read` from True to False to also wait writes
-                                    read=False,
+                            reg_out_ptrs = cute.make_tensor(
+                                smem_out_ptrs_ptr, (8,)
+                            ).load()
+                            for iter in cutlass.range_constexpr(8):
```

Ruff (0.15.7)

[error] Variable `iter` is shadowing a Python builtin

(A001)